src/store/tracks.js

```diff
@@ -23,6 +23,25 @@ export default {
     removeTrack(state, id) {
       Vue.delete(state.tracks, id);
     },
+    updateGenreOccurence(state, { newID, oldID }) {
+      for (const t in state.tracks) {
+        const i = state.tracks[t].genre_ids.findIndex(
+          (gId) => `${gId}` === `${oldID}`
+        );
+        if (i >= 0) {
+          state.tracks[t].genre_ids.splice(i, 1);
+          // if newID isn't provided we'll just remove the old genre
```

Comment:
I feel like we shouldn't handle this case? If newID isn't provided, it's a programmer error to use the function in the wrong way.

Reply:
This way, we can use this function after a genre merge and after a genre is deleted (see #77 )
I haven't implemented that yet, since that seemed like something for a different PR

Reply:
If that is something we want to do, I feel like it should be a different function or this one should be renamed (when it is implemented). Until then, it doesn't make sense to me to already handle this.

Reply:
Sure